go/internal/feast/featurestore.go

```diff
@@ -622,30 +621,11 @@ func (fs *FeatureStore) transposeResponseIntoColumns(featureData2D [][]FeatureDa
 				currentVector.Timestamps[rowIndex] = eventTimeStamp
 			}
 		}
-		var fieldType arrow.DataType
-		var err error
-
-		for _, val := range protoValues {
-			if val != nil {
-				fieldType, err = utils.ProtoTypeToArrowType(val)
-				if err != nil {
-					return nil, err
-				}
-				break
-			}
-		}
-
-		if fieldType != nil {
-			builder := array.NewBuilder(arrowAllocator, fieldType)
-			err = utils.ProtoValuesToArrowArray(builder, protoValues)
-			if err != nil {
-				return nil, err
-			}
-
-			currentVector.Values = builder.NewArray()
-		} else {
-			currentVector.Values = array.NewNull(numRows)
+		arrowValues, err := utils.ProtoValuesToArrowArray(protoValues, arrowAllocator, numRows)
```

Comment:
nit, can we have a different module name instead of `utils`? Seems like it'll become a kitchen sink..

Reply:
renamed into `types` since it's type utils